.github/workflows/agw-workflow.yml

```diff
@@ -213,6 +224,12 @@ jobs:
         with:
           name: Unit Test Results
           path: c-cpp-test-results/
+      - name: Build space left after run
+        shell: bash
+        run: |
+          echo "Available storage:"
+          df -h
+          echo
```

Comment:
Nit: What are the empty `echo`s needed for - is a newline needed?

Reply:
No, they are not needed. I used a couple for esthetic reasons - and removed them. forgot about these. removed.